packages/salesforcedx-vscode-aura/package.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "salesforcedx-vscode-aura",
+  "displayName": "Lightning",
+  "description": "Provides code-editing features for Lightning",
+  "qna": "https://github.com/forcedotcom/salesforcedx-vscode/issues",
+  "bugs": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode/issues"
+  },
+  "repository": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode"
+  },
+  "icon": "images/VSCodeLWC.png",
+  "galleryBanner": {
+    "color": "#ECECEC",
+    "theme": "light"
+  },
+  "version": "45.6.0",
+  "publisher": "salesforce",
+  "preview": true,
+  "license": "BSD-3-Clause",
+  "engines": {
+    "vscode": "^1.26.0"
+  },
+  "categories": [
+    "Programming Languages"
+  ],
+  "dependencies": {
+    "@salesforce/salesforcedx-utils-vscode": "45.6.0",
+    "chalk": "^2.4.2",
+    "change-case": "^3.1.0",
+    "lightning-lsp-common": "1.0.4",
+    "lwc-language-server": "1.8.3",
+    "opn": "^5.4.0",
+    "vscode": "1.1.17",
+    "vscode-extension-telemetry": "0.0.17",
+    "vscode-languageclient": "^3.5.1"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.0.0",
+    "@types/mocha": "2.2.38",
+    "@types/opn": "^5.1.0",
+    "@types/node": "8.9.3",
+    "@types/sinon": "^2.3.2",
+    "typescript": "3.1.6",
+    "chai": "^4.0.2",
+    "cross-env": "^5.0.5",
+    "mocha": "3.2.0",
+    "sinon": "^2.3.6"
+  },
+  "scripts": {
+    "vscode:prepublish": "npm prune --production",
+    "vscode:package": "vsce package",
+    "compile": "tsc -p ./",
+    "lint": "tslint --project .",
+    "watch": "tsc -watch -p .",
+    "clean": "shx rm -rf node_modules && shx rm -rf out && shx rm -rf coverage && shx rm -rf .nyc_output",
+    "postinstall": "node node_modules/.bin/vscode-install",
+    "pretest": "node ../../scripts/download-vscode-for-system-tests",
+    "test": "node ../../scripts/install-vsix-dependencies dbaeumer.vscode-eslint && node ../../scripts/run-test-with-top-level-extensions",
+    "test:insiders": "node ../../scripts/install-vsix-dependencies dbaeumer.vscode-eslint && cross-env CODE_VERSION=insiders node ../../scripts/run-test-with-top-level-extensions",
+    "test:unit": "node ../../scripts/install-vsix-dependencies dbaeumer.vscode-eslint && node ../../scripts/run-test-with-top-level-extensions"
+  },
+  "activationEvents": [
+    "workspaceContains:sfdx-project.json",
+    "workspaceContains:**/core/workspace-user.xml"
```

Comment:
Where is `/core/workspace-user.xml` generated from ? Is it so it works for internal devs as well ?

Reply:
Yes, this is strictly an internal dev thing